integration/spark/src/main/scala/org/apache/spark/sql/index/CarbonIndexUtil.scala

```diff
@@ -377,4 +381,212 @@ object CarbonIndexUtil {
       AlterTableUtil.releaseLocks(locks.asScala.toList)
     }
   }
+
+  def processSIRepair(indexTableName: String, carbonTable: CarbonTable,
+    carbonLoadModel: CarbonLoadModel, indexMetadata: IndexMetadata,
+      mainTableDetails: List[LoadMetadataDetails], secondaryIndexProvider: String)
+  (sparkSession: SparkSession) : Unit = {
+    val sparkSession = SparkSession.getActiveSession.get
```

Comment:
please use the sparkSession which is passed

Reply:
done